src/CoreWCF.Primitives/src/CoreWCF/Security/MessageCredentialType.cs

```diff
@@ -14,7 +14,7 @@ public enum MessageCredentialType
 
     internal static class MessageCredentialTypeHelper
     {
-        internal static bool IsDefined(MessageCredentialType value)
+        public static bool IsDefined(MessageCredentialType value)
         {
```

Comment:
Although it doesn't change visibility as the class is internal, can you revert this back to internal again.